### DISCUSSION TITLE
Can anyone recommend something similar for Microsoft Windows?

### DISCUSSION THREAD

**Discussion by caban** [rxhanson/Rectangle#835] [Q&A]
I'm loving this on my Mac and want something that acts exactly the same on Microsoft Windows.

**Comment by rxhanson** [comment]
Maybe this is what you want? (I'm not affiliated)
https://github.com/ahmetb/RectangleWin

Discussed here previously (https://github.com/rxhanson/Rectangle/discussions/752)
